statsmodels/regression/tests/test_lme.py

```diff
@@ -808,18 +854,17 @@ def do1(reml, irf, ds_ix):
 fnames = os.listdir(rdir)
 fnames = [x for x in fnames if x.startswith("lme") and x.endswith(".csv")]
 
-import itertools
-import nose.tools
-import pytest
 
 # Copied from bashtage's #3847
 @nose.tools.nottest
 @pytest.mark.parametrize('fname,reml,irf',
-                         itertools.product(fnames, [False, True], [False, True]))
+                         itertools.product(fnames, [False, True],
+                                           [False, True]))
```

Comment:
As long as this is being futzed with, the idiomatic way to do this would be

```
@pytest.mark.parametrize('fname', fnames)
@pytest.mark.parametrize('reml', [False, True])
@pytest.mark.parametrize('irf', [False, True])
def test_r(fname, reml, irf):
```